EateryModel/Sources/EateryModel/Networking/EateryAPI.swift

```diff
@@ -31,5 +31,28 @@ public struct EateryAPI {
         return schemaApiResponse.data.map(SchemaToModel.convert)
     }
 
+    public func reportError(eateryID: Int, type: String, content: String) -> Void {
```

Comment:
```suggestion
    public func reportError(eateryID: Int, type: String, content: String) {
```

Reply:
Can we also make this an async method?

Reply:
Making this async makes a long list of methods async it would then become
- submit() becomes async
- submitButton.tap { [self] _ in becomes async
- addSubmitButton() would become async
- setUpView() would become async

Is there a way to make this method async without making everything else async?

Reply:
Use `Task` (see https://github.com/cuappdev/eatery-blue-ios/blob/266d7ceeccd4d53498309a0cb8dbeed0f6081468/Eatery%20Blue/Eatery%20Blue/UI/HomeScreen/HomeModelController.swift#L29)

Reply:
This is also a handy reference: https://docs.swift.org/swift-book/LanguageGuide/Concurrency.html#ID642